src/terminal/terminal/terminal.h

```diff
@@ -97,6 +97,16 @@
  */
 #define GUAC_TERMINAL_WHEEL_SCROLL_AMOUNT 3
 
+/**
+ * The size of margins between the console text and the border in pixels.
+ */
+#define GUAC_TERMINAL_MARGINS 16
```

Comment:
This and `GUAC_TERMINAL_OFFSET` will need to be in some pixel-independent units to allow the margins to scale for different display resolutions. Not all screens will have the same DPI, and the terminal font rendering takes this into account.